src/compiler/checker.ts

```diff
@@ -15201,15 +15201,15 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
         return true;
     }
 
-    function getSignatureOfTypeTag(node: SignatureDeclaration | JSDocSignature) {
+    function getSignatureOfTypeTag(node: SignatureDeclaration | JSDocSignature, allowMembers?: boolean) {

```

Comment:
how many callers are there of these two functions? Could allowMembers be required without changing tonnes of call sites?